docs/pages/fund-agents-apps/configure-agent.mdx

```diff
@@ -0,0 +1,63 @@
+# Configure your agent
+
+Your agent needs to pay for the messages it sends on the XMTP network. To get started, you'll need.
+
+1. An agent built using our Node or Agent SDK
+2. A payer wallet that has been [funded through the XMTP Funding Portal](./fund-your-app.mdx)
+
+## Setup your Signer
+
+When you create a client, you already specify a `Signer` that is used to link your XMTP identity to a wallet. You can choose to use the same `Signer` to pay for messages, or create a new one purely for this purpose.
+
+This Signer must be of type `eoa`. The `messagePayer` may not be a smart contract wallet, since the signatures need to be able to be verified offchain.
```

Comment:
```suggestion
This Signer must be of type `eoa`. The `messagePayer` may not be a smart contract wallet, since the signatures need to be able to be verified offchain.

:::info

You can fund your payer wallet from a smart contract wallet [using the XMTP Funding Portal](./fund-your-app.mdx).

:::